pmda/rms/__init__.py

```diff
@@ -0,0 +1,29 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# PMDA
+# Copyright (c) 2019 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+
+"""
+pmda.rms
+========
+
+Ready to use root-mean-square analyses built via MDAnalysis with dask.
+
+The full documentation can be found at https://www.mdanalysis.org/pmda/.
+
+Note that by default no modules are imported in the top-level name
```

Comment:
I don't understand this sentence. We _are_ importing `RMSD` and `RMSF` into `rms` in the lines below (which is how it should be).

Reply:
As far as I can tell, this is an artifact carried over from MDAnalysis that I didn't remove. I'll just take it (and the following lines) out.